src/test/resources/org/jenkinsci/plugins/credentialsbinding/impl/CertificateMultiBindingTest/basicsPipeline-step1.bat

```diff
@@ -9,3 +9,4 @@ if not defined KEYSTORE_ALIAS exit /B 1
 REM keep location of the keystore file for the next step
 echo %MY_KEYSTORE% > keystore-path
 
+exit /B 0
```

Comment:
Hi @jglick,

As I can see from log of the previous commit https://github.com/jenkinsci/credentials-binding-plugin/runs/2889451321, I'd recommend you to add quotes around `%MY_KEYSTORE%` on line 3
```bat
if not exist "%MY_KEYSTORE%" exit /B 1
```
The problem is happened because `cmd.exe` doesn't reset its `%errorlevel%` after executing internal commands like `if` or `echo`.

Reply:
@dsolovyev if you have a moment could you check 65272cb?

Reply:
@jglick, it looks good to me .